src/dotty/tools/dotc/core/Comments.scala

```diff
@@ -2,8 +2,12 @@ package dotty.tools
 package dotc
 package core
 
-import ast.{ untpd, tpd }
-import Decorators._, Symbols._, Contexts._, Flags.EmptyFlags
+import ast.{tpd, untpd}
+import Decorators._
+import Symbols._
+import Contexts._
+import Flags.EmptyFlags
+import dotty.tools.dotc.reporting.diagnostic.messages.ProperDefinitionNotFound
```

Comment:
I think a lot of these changes can be left out, basically the only addition here should be something like:

```scala
// you should be able to omit most of the path and get something like:
import reporting.diagnostic.messages.ProperDefinitionNotFound
// or even just:
import reporting.diagnostic.messages._
```

whichever floats your goat :)